src/WCSimEnumerations.cc

```diff
@@ -143,6 +143,39 @@ std::string WCSimEnumerations::EnumAsString(ProcessType_t p)
   case (kNeutronInelastic) :
     return "neutronInelastic";
     break;
+  case (kMuMinusCaptureAtRest) :
+    return "kMuMinusCaptureAtRest";
```

Comment:
Should this return "muMinusCaptureAtRest" not "kMuMinusCaptureAtRest"?

Reply:
Good eyes. Thanks @nickwp